js/components/utils.js

```diff
@@ -410,6 +410,27 @@ var Utils = {
 
   },
 
+    /**
+  * Get the countdown string that shows the time until a given future timestamp
+  *
+  * @function getStartCountdown
+  * @param {Number} timestamp - The Unix timestamp for the asset flight time start
+  * @returns {String} The countdown time string
+  */
+  getStartCountdown: function(countdown_timestamp) {
```

Comment:
Can we change countdown_timestamp to countdownTimestamp to match the styling for the rest of this file?

Reply:
@aeng7 fixed